package.json

```
@@ -12,7 +12,9 @@
         "preview": "vite preview",
         "prepare": "husky install",
         "json-db": "json-server --watch db.json",
-        "dev-db": "concurrently \"npm:json-db\" \"npm:dev\""
+        "dev-db": "concurrently \"npm:json-db\" \"npm:dev\"",
+        "extract": "formatjs extract",
+        "compile": "formatjs compile"
```

I think you need to change it like this:

```suggestion
        "extract": "formatjs extract \"src/**/*.{ts,tsx}\" --ignore \"**/*.d.ts\"",
        "compile": "formatjs compile \"src/**/*.{ts,tsx}\" --ignore \"**/*.d.ts\""
```
otherwise I get an error when running the commands
Also maybe you need to specify the output files for the languages I am not sure?

---

Added specific scripts for the different supported languages